elizabeth/data/fa/science.json

```diff
@@ -0,0 +1,214 @@
+    "https://fa.wikipedia.org/wiki/%D8%B3%D8%B1%D9%85%D8%A7%D8%AE%D9%88%D8%B1%D8%AF%DA%AF%DB%8C",
+    "https://fa.wikipedia.org/wiki/%D9%81%DB%8C%D8%B2%DB%8C%DA%A9_%D9%BE%D8%B2%D8%B4%DA%A9%DB%8C",
+    "https://fa.wikipedia.org/wiki/%D8%B3%DB%8C%D9%81%D9%84%DB%8C%D8%B3",
+    "https://fa.wikipedia.org/wiki/%D8%AC%D9%88_%D8%B2%D9%85%DB%8C%D9%86",
+    "https://fa.wikipedia.org/wiki/%D8%B2%D9%85%DB%8C%D9%86%E2%80%8C%D9%84%D8%B1%D8%B2%D9%87_%D9%82%D8%A7%D8%A6%D9%86_(%DB%B1%DB%B3%DB%B7%DB%B6)",
+    "https://fa.wikipedia.org/wiki/%D8%A7%DA%98%D8%AF%D9%87%D8%A7%DB%8C_%DA%A9%D9%88%D9%85%D9%88%D8%AF%D9%88",
+    "https://fa.wikipedia.org/wiki/%D9%BE%D9%86%DA%AF%D9%88%D8%A6%D9%86_%D8%A7%D9%85%D9%BE%D8%B1%D8%A7%D8%AA%D9%88%D8%B1",
+    "https://fa.wikipedia.org/wiki/%D8%AF%D9%84%D9%81%DB%8C%D9%86",
+    "https://fa.wikipedia.org/wiki/%D8%B1%D8%B2",
+    "https://fa.wikipedia.org/wiki/%D8%B1%D9%81%D8%AA%D8%A7%D8%B1_%D9%87%D9%85%D8%AC%D9%86%D8%B3%E2%80%8C%DA%AF%D8%B1%D8%A7%DB%8C%D8%A7%D9%86%D9%87_%D8%AF%D8%B1_%D8%AC%D8%A7%D9%86%D9%88%D8%B1%D8%A7%D9%86",
+    "https://fa.wikipedia.org/wiki/%D8%B3%D8%A7%DA%A9%D9%88%D8%B1%D8%A7",
+    "https://fa.wikipedia.org/wiki/%D8%B4%DB%8C%D8%B1_(%DA%AF%D8%B1%D8%A8%D9%87%E2%80%8C%D8%B3%D8%A7%D9%86)",
+    "https://fa.wikipedia.org/wiki/%D8%B4%DB%8C%D8%B7%D8%A7%D9%86_%D8%AA%D8%A7%D8%B3%D9%85%D8%A7%D9%86%DB%8C",
+    "https://fa.wikipedia.org/wiki/%D9%81%D8%B1%DA%AF%D8%B4%D8%AA_%D8%A2%D8%A8%E2%80%8C%D8%A8%D8%A7%D8%B2%D8%B3%D8%A7%D9%86%D8%A7%D9%86",
+    "https://fa.wikipedia.org/wiki/%D9%82%D9%88",
+    "https://fa.wikipedia.org/wiki/%DA%A9%DB%8C%D8%B3%D9%87%E2%80%8C%D8%AF%D8%A7%D8%B1%D8%A7%D9%86",
+    "https://fa.wikipedia.org/wiki/%D9%84%D8%A7%D9%84%D9%87",
+    "https://fa.wikipedia.org/wiki/%D9%86%D9%87%D9%86%DA%AF_%D9%82%D8%A7%D8%AA%D9%84",
+    "https://fa.wikipedia.org/wiki/%DB%8C%D9%88%D8%B2%D9%BE%D9%84%D9%86%DA%AF",
+    "https://fa.wikipedia.org/wiki/%D8%B1%D8%A2%DA%A9%D8%AA%D9%88%D8%B1_%D8%B4%DB%8C%D9%85%DB%8C%D8%A7%DB%8C%DB%8C",
+    "https://fa.wikipedia.org/wiki/%D9%81%D8%B1%D8%A7%D9%86%D8%B3%DB%8C%D9%85",
+    "https://fa.wikipedia.org/wiki/%D8%A7%D8%B1%DB%8C%D8%B3",
+    "https://fa.wikipedia.org/wiki/%D8%A7%D9%84%DA%A9%D8%B3%DB%8C_%D9%84%D8%A6%D9%88%D9%86%D9%88%D9%81",
+    "https://fa.wikipedia.org/wiki/%D9%85%D8%A7%DA%A9%DB%8C%D8%A7%D9%86_%D8%A7%DB%8C%DA%A9%D8%B3_%DB%8C%DA%A9",
+    "https://fa.wikipedia.org/wiki/%DB%8C%D9%88%D8%B1%DB%8C_%DA%AF%D8%A7%DA%AF%D8%A7%D8%B1%DB%8C%D9%86",
+    "https://fa.wikipedia.org/wiki/%D8%B3%D8%A7%D9%85%D8%A7%D9%86%D9%87_%D9%87%D8%B4%D8%AF%D8%A7%D8%B1_%D8%B2%D9%85%DB%8C%D9%86%E2%80%8C%D9%84%D8%B1%D8%B2%D9%87",
+    "https://fa.wikipedia.org/wiki/%D9%85%D8%AE%D8%B2%D9%86_%D8%AE%D8%A7%D8%B1%D8%AC%DB%8C_%D8%B4%D8%A7%D8%AA%D9%84_%D9%81%D8%B6%D8%A7%DB%8C%DB%8C",
+    "https://fa.wikipedia.org/wiki/%D9%86%DB%8C%D8%B1%D9%88%DA%AF%D8%A7%D9%87_%D9%87%D8%B3%D8%AA%D9%87%E2%80%8C%D8%A7%DB%8C"
+  ],
+  "chemical_element": [
```

Comment:
@lk-geimfari I haven't been able to spend much time recently, I guess this will work fine for now, and I'll add new data soon. Is there a reason why this has not been merged?